deployer/pkg/context/infraContext.go

```diff
@@ -0,0 +1,34 @@
+package context
+
+import (
+	"log"
+	"os"
+	"path"
+)
+
+const (
+	ClustersFolder = "clusters"
+)
+
+type InfraContext struct {
+	ScriptPath          string
+	AnsiblePlaybookPath string
+	SourceClustersDir   string
+	InfraRootDir        string
+}
+
+func InitInfraContext() *InfraContext {
+	rootDir, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("Cannot extract current working directory from environment, detailed error:  %s", err)
+		return nil
+	}
+	ic := InfraContext{
+		ScriptPath:          path.Join(rootDir,"infra/parser.sh") ,
+		AnsiblePlaybookPath: path.Join(rootDir,"infra/playbook"),
```

Comment:
I can't see the playbook folder in the commit, maybe you missed to add it?

Reply:
I Didn't missed that..
It's on my computer, And i still wonder if to add it as a copy of the ocp_lab_provisioner repo (playbook repo) or to add that repo as a git submodule.